charts/tempo-distributed/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: tempo-distributed
 description: Grafana Tempo in MicroService mode
 type: application
-version: 1.4.13
-appVersion: 2.1.1
+version: 1.5.0
+appVersion: 2.2.0-rc.0
```

Comment:
Do we want the RC?  I think if we wait a week or so, we'll have a proper release.

Reply:
I guess I was a little eager to try out the new RC :grin: 

Happy to wait for the official release, or do a separate PR for it - either is fine with me.

Reply:
I think wait maybe two days and we'll have a release.  But by all means, test out the image with these changes if you can.

Reply:
Updated :tada:

Reply:
@joe-elliott put up https://github.com/grafana/helm-charts/pull/2562 for review.  See some slight differences between the two.

Reply:
Looks like this PR is a superset of Joe's. Both PRs upgrade to 2.2.0 and remove the deprecated `tolerate_failed_blocks`, but this PR also adds support for the new `querier.config.search.external_backend` feature.

Reply:
Happy to followup with adding `querier.config.search.external_backend` in a followup PR, or we can just merge this one in to have it all done at once.

Reply:
@zalegrala I rebased this PR so that it only updates `querier.config.search.external_backend`, now that the v2.2.0 changes have been merged.

Reply:
Nice.  Thanks for the updates.